content/blog/CDKTF-what-next/index.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "CDKTF is deprecated: What's next for your team?"
+date: 2025-12-18
+draft: false
+meta_desc: "CDKTF is deprecated. Here are your options: stay on CDKTF, move to Terraform/OpenTofu, or migrate to Pulumi and keep writing infrastructure in real languages."
+meta_image: cdktf-archived.png
+authors:
+  - adam-gordon-bell
+  - christian-nunciato
+tags:
+  - migration
+  - terraform
+---
+
+In July, 2020, CDK for Terraform (CDKTF) was introduced, and last week, on December 10, it was officially deprecated. Support for CDKTF has stopped, the organization and repository [have been archived](https://github.com/cdktf), and HashiCorp/IBM will no longer be updating or maintaining it, leaving a lot of teams out there without a clear path forward.
+
+For most teams, that means it's time to start looking for a replacement.
```

Comment:
Consider "organizations" instead of "teams". The former is probably more commercially oriented, the latter closer to top of funnel. I assume the choice was intentional tho.